_assets/styles/scss/04-components/_forms--comment.scss

```diff
@@ -154,3 +147,105 @@ parent: forms
     border-width: 3px;
   }
 }
+
+// The entire text editor component.
```

Comment:
@AnneTee is there any value in splitting this out into its own `__forms--comment--editor.scss` file?